liquibase-service/src/main/resources/db/odse/routines/010-sp_investigation_event-002.sql

```diff
@@ -767,12 +792,33 @@ BEGIN
-                              MAX(CASE WHEN type_cd = 'OrgAsClinicOfPHC' THEN entity_uid END)   ordering_facility_uid
+             (SELECT DISTINCT
+                  act_uid  AS  nac_page_case_uid,
+                  MAX(last_chg_time) AS                                                  nac_last_chg_time,
+                  MIN(add_time)      AS                                                 nac_add_time,
+                  MAX(CASE WHEN type_cd = 'InvestgrOfPHC' THEN entity_uid END) INVESTIGATOR_uid,
+                  MAX(CASE WHEN type_cd = 'PerAsReporterOfPHC' THEN entity_uid END) PERSON_AS_REPORTER_UID,
+                  MAX(CASE WHEN type_cd = 'SubjOfPHC' THEN entity_uid END) patient_uid,
+                  MAX(CASE WHEN type_cd = 'PhysicianOfPHC' THEN entity_uid END) PHYSICIAN_UID,
+                  MAX(CASE WHEN type_cd = 'CASupervisorOfPHC' THEN entity_uid END) CA_SupervisorOfPHC,
+                  MAX(CASE WHEN type_cd = 'ClosureInvestgrOfPHC' THEN entity_uid END) Closure_Investgr_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'DispoFldFupInvestgrOfPHC' THEN entity_uid END) Dispo_Fld_FupInvestgr_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'FldFupInvestgrOfPHC' THEN entity_uid END) Fld_Fup_Investgr_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'FldFupProvOfPHC' THEN entity_uid END) Fld_Fup_Prov_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'FldFupSupervisorOfPHC' THEN entity_uid END) Fld_Fup_Supervisor_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'InitFldFupInvestgrOfPHC' THEN entity_uid END) Init_Fld_Fup_Investgr_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'InitFupInvestgrOfPHC' THEN entity_uid END) Init_Fup_Investgr_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'InitInterviewerOfPHC' THEN entity_uid END) Init_Interviewer_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'InterviewerOfPHC' THEN entity_uid END) Interviewer_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'SurvInvestgrOfPHC' THEN entity_uid END) Surv_Investgr_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'FldFupFacilityOfPHC' THEN entity_uid END) Fld_Fup_Facility_Of_PHC,
+                  MAX(CASE WHEN type_cd = 'HospOfADT' THEN entity_uid END) HOSPITAL_UID,
+                  MAX(CASE WHEN type_cd = 'OrgAsHospitalOfDelivery' THEN entity_uid END) Org_As_Hospital_Of_Delivery,
+                  MAX(CASE WHEN type_cd = 'PerAsProviderOfDelivery' THEN entity_uid END) Per_As_Provider_Of_Delivery,
+                  MAX(CASE WHEN type_cd = 'PerAsProviderOfOBGYN' THEN entity_uid END) Per_As_Provider_Of_OBGYN,
+                  MAX(CASE WHEN type_cd = 'PerAsProvideroOfPediatrics' THEN entity_uid END) Per_As_Provider_Of_Pediatrics,
+                  MAX(CASE WHEN type_cd = 'OrgAsReporterOfPHC' THEN entity_uid END) ORG_AS_REPORTER_UID,
+                  MAX(CASE WHEN type_cd = 'OrgAsClinicOfPHC' THEN entity_uid END) ordering_facility_uid
```

Comment:
Is DISTINCT needed here? Just asking because it's a lot of columns, I personally don't know the data well enough to know whether or not this would be guaranteed to produce a unique result set without the DISTINCT keyword.

Reply:
I'll roll this back for now since it's part of a bigger update to the sproc- not intended for this PR.
@johneipe

Reply:
Ok sure, sounds good